pkg/storage/ingest/reader.go

```diff
@@ -581,6 +581,8 @@ func (r *partitionCommitter) stop(error) error {
 
 type readerMetrics struct {
 	receiveDelay              *prometheus.HistogramVec
```

Comment:
I would remove it from here, given it should be never directly used.

Reply:
Removed.